docs/docs/getting-started/first-steps.md

```diff
@@ -152,6 +159,8 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
     Fill the values produced by the script into your configuration file.
 
+    Please note that only regions which offer CVMs of the `N2D` series mentioned below are supported. Find a full list of regions [here](https://cloud.google.com/compute/docs/regions-zones). To see which regions offer the `N2D` series, filter by machine type: `N2D`. Alternatively, you can run `gcloud compute machine-types list --filter="name~n2d" --format="table[box,title='N2D series'](name,zone,description)"` to list all available machine types.
```

Comment:
Can you also add this hint to the "GCP (Console)" tab to the *regions* field? Maybe omit the gcloud command there.

Reply:
Omitting the gcloud command was merely meant for the "GCP (Console)" tab as you don't interact with the shell there. I'd be fine with including it in the "GCP (CLI)" tab. I'm also fine with omitting it. Up to you.